src/rws_common.cpp

```diff
@@ -101,20 +101,20 @@ std::string xmlFindTextContent(const Poco::XML::Node* p_node, const XMLAttribute
       bool found = false;
       if (p_node->hasChildNodes())
       {
-        Poco::XML::NodeList* children = p_node->childNodes();
+        Poco::AutoPtr<Poco::XML::NodeList> p_children(p_node->childNodes());
```

Comment:
At the moment we're not, but it would be nice if we could switch to C++11. That would remove the need for custom classes like `Poco::AutoPtr<>` for things like `shared_ptr`s.

Reply:
I have added it to my TODO-list and I will open an issue for it.